src/open_inwoner/configurations/tests/test_analytics.py

```diff
@@ -111,3 +115,57 @@ def test_siteimprove_analytics_not_rendered(self):
 
         self.assertTemplateNotUsed(response, "analytics/siteimprove.html")
         self.assertNotContains(response, "<!-- SiteImprove -->")
+
+
+@tag("e2e")
+@override_settings(ROOT_URLCONF="open_inwoner.cms.tests.urls")
+class CookieBannerPlaywrightTests(PlaywrightSyncLiveServerTestCase):
+    def setUp(self):
+        from open_inwoner.accounts.tests.factories import DigidUserFactory
+
+        super().setUp()
+
+        self.user = DigidUserFactory.create()
+        self.user_login_state = self.get_user_bsn_login_state(self.user)
+
+    def test_cookie_accept(self):
+        config = SiteConfiguration.get_solo()
```

Comment:
If you use a Solo object like this it is safer to also use the `ClearCachesMixin` so it doesn't leak data across tests. 

Alternately you can use the patch/mock solution (eg: `@patch("open_inwoner.configurations.models.SiteConfiguration.get_solo")` etc) to get a clean instance (although I had weird issue with text fields using this so used the regular get/save).

Example: it is not clear if there would be a `config.cookie_info_text` set here even though it is the default of the model.

Reply:
You're right about the second approach (I'd have to explicitly set the value of `config.cookie_info_text`), so I'll stick with the mixin